functions/palindrome_number.py

```diff
@@ -1,2 +1,12 @@
 def palindrome_number(x):
-    pass
+    temp = x
+    reverse = 0
+    while(x>0):
```

Comment:
Python's style guide recommends always surrounding binary operators with a single space on either side, like this: `while(x > 0):`

[https://peps.python.org/pep-0008/#other-recommendations](url)